packages/zoe/test/unitTests/contracts/test-zcf.js

```diff
@@ -27,7 +28,43 @@ test('zoe - test zcf', async t => {
     Pixels: moolaIssuer,
     Money: simoleanIssuer,
   });
-  await t.notThrowsAsync(() =>
-    zoe.startInstance(installation, issuerKeywordRecord),
+
+  const { creatorFacet } = await E(zoe).startInstance(
+    installation,
+    issuerKeywordRecord,
   );
+
+  // This contract gives ZCF as the contractFacet for testing purposes
+  /** @type ContractFacet */
+  const zcf = creatorFacet;
+
+  let firstSeat;
+
+  const grabSeat = seat => {
+    firstSeat = seat;
+    return 'ok';
+  };
+
+  const kickOutSeat = secondSeat => {
+    firstSeat.kickOut(new Error('kicked out first'));
+    throw secondSeat.kickOut(new Error('kicked out second'));
+  };
+
+  const invitation1 = zcf.makeInvitation(grabSeat, 'seat1');
+  const invitation2 = zcf.makeInvitation(kickOutSeat, 'seat2');
+
+  const userSeat1 = await E(zoe).offer(invitation1);
+  const userSeat2 = await E(zoe).offer(invitation2);
+  const userSeat1Result = await E(userSeat1).getOfferResult();
+
+  t.is(userSeat1Result, 'ok');
+
+  await E(userSeat2).getPayouts();
+  // Results in "Unhandled rejection"
+  // E(userSeat2).getOfferResult();
+
+  await t.throwsAsync(() => E(userSeat2).getOfferResult());
```

Comment:
This currently isn't being run, but I can't figure out why. I get "Planned for 1 assertion, but got 3."